packages/base/src/3dview/mainview.tsx

```diff
@@ -619,6 +621,14 @@ export class MainView extends React.Component<IProps, IStates> {
             this._selectedMeshes.push(el as any as BasicMesh);
             el.material.color = SELECTED_MESH_COLOR;
             el.material.linewidth = SELECTED_LINEWIDTH;
+            el.userData.originalColor = el.material.color.clone();
```

Comment:
Shouldn't it be the following? I thought `el.userData.originalColor` was used to store the mesh color as defined in the model?

```suggestion
            el.userData.originalColor = new THREE.Color(objColor);
```

Reply:
yes, my bad. Sorry.